diff --git a/dist/chunk-7TKX73CW.js b/dist/chunk-7TKX73CW.js
index 84f449f1c37310ac8fb35f9e4b996047940fa01e..ceb34e9b5f5fe47ba7e422a0822c70c731b90c15 100644
--- a/dist/chunk-7TKX73CW.js
+++ b/dist/chunk-7TKX73CW.js
@@ -2064,7 +2064,7 @@ async function getMarkdownTransformers(options) {
     transformPlantUml,
     options.data.features.monaco && transformMonaco,
     ...extras.postCodeblock,
-    transformCodeWrapper,
+    // transformCodeWrapper,
     options.data.features.katex && transformKaTexWrapper,
     transformPageCSS,
     transformSlotSugar,
